packages/components/tabs/src/tab.tsx

```diff
@@ -61,6 +61,10 @@ const Tab = forwardRef<"button", TabItemProps>((props, ref) => {
     isPressed,
   } = useTab({key, isDisabled: isDisabledProp, shouldSelectOnPressUp}, state, domRef);
 
+  if (props.children == null) {
+    delete tabProps["aria-controls"];
+  }
```

Comment:
When [displaying tabs only](https://nextui.org/docs/components/tabs#colors), like in a catalog, `aria-controls` is not set.